ansible_base/lib/jwt_auth/hub/auth.py

```diff
@@ -0,0 +1,15 @@
+from ansible_base.lib.jwt_auth.common.auth import JWTAuthentication
+from ansible_base.lib.jwt_auth.common.exceptions import InvalidService
+
+
+class HubJWTAuth(JWTAuthentication):
```

Comment:
Interesting. I expected this sort of code to live in hub, not in the shared library.